libmscore/navigate.cpp

```diff
@@ -35,16 +36,13 @@ namespace Ms {
 //    return next Chord or Rest
 //---------------------------------------------------------
 
-ChordRest* nextChordRest(ChordRest* cr, bool skipGrace)
+ChordRest* nextChordRest(ChordRest* cr, bool skipGrace, bool skipMeasureRepeatRests)
```

Comment:
Curious that this was needed here, Ordinary navigation (left/right) seems to still hit the hidden rests, though.  Could you explain more about what should and should not be happening here?

Reply:
On my setup ordinary navigation with the arrow keys does not hit the hidden rests. Could you possibly have assigned the shortcuts to similar but different actions?